client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -6,36 +6,90 @@ export type AllowedTypes = 'site' | 'backup' | 'scan' | 'monitor' | 'plugin';
 // Site column object which holds key and title of each column
 export type SiteColumns = Array< { key: string; title: ReactChild } >;
 
-export type AllowedStatusTypes = 'inactive' | 'progress' | 'failed' | 'warning' | 'success';
+export type AllowedStatusTypes =
+	| 'inactive'
+	| 'progress'
+	| 'failed'
+	| 'warning'
+	| 'success'
+	| 'disabled';
 
+export interface SiteObj {
```

Comment:
It's better to name it as `Site` instead of `SiteObj`

```suggestion
export interface Site {
```

Reply:
I agree with this one, the `Obj` (as similar to `*Arr`) isn't bringing any additional context, so it's better to avoid it.

Reply:
@manzoorwanijk, @vitozev - Thanks for the feedback. I have made the changes.